File: livetester-ui/src/shared/vo/element-config-vo.ts
export class ElementConfigVO {
    id: number;
    label: string;
    fieldName: string;
    fieldType: string;
    controlType: string;
    mandatory: string;
    applicable: string;
    matchQuery: string;
    fallbackQuery1: string;
    fallbackQuery2: string;
    elementLabel: string;
    json: any;
}

export class ElementConfigListVO {
    id: number;
    labelNames: string;
    empty: '';
    fieldName: string;
    fieldType: string;
    isMandatory: string;
    applicable: string;
    matchQuery: string;
    fallbackQuery1: string;
    fallbackQuery2: string;
    elementLabel: string;
    controlType: string;
    json: any;
}

File: livetester-service/src/main/java/com/yorosis/livetester/vo/ElementConfigVO.java
package com.yorosis.livetester.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ElementConfigVO {
	private Long id;

	private String label;
	private String fieldName;

	private String fieldType;
	private String controlType;
	private String json;
	private String mandatory;
	private String applicable;
	private String matchQuery;
	private String fallbackQuery1;
	private String fallbackQuery2;
}
